oracle/datadog_checks/oracle/oracle.py

```diff
@@ -114,6 +123,13 @@ def create_connection(self):
         try:
             if use_oracle_client:
                 connection = cx_Oracle.connect(connect_string)
+            elif JDBC_IMPORT_ERROR:
+                self.log.error(
+                    "Oracle client is unavaible and the integration is unable to import JDBC libraries. You may not"
```

Comment:
Is it unavailable or not selected?

Reply:
Both actually. If oracle client is selected, the check will work no matter what and won't log any error with JDBC libraries.

If oracle client is not selected *and* the check is unable to import jpype/jaydebeapi, the ImportError exception will be raised.

Reply:
Ohh, I see, I thought use_oracle_client was a config option